invenio_cli/cli/containers.py

```diff
@@ -21,9 +21,9 @@ def containers():
 
 @containers.command()
 @click.option('--pull/--no-pull', default=True, is_flag=True,
-              help='Download newer versions of the images (default: True).')
+              help='Download newer versions of the images (default: Download).')
```

Comment:
what about
```suggestion
              help='Download/or not newer versions of the images (default: pull).')
```

What I am afraid is that is 100% clear if download means pull or no pull. Still Im not 100% sure about my suggestion.